src/main/java/core/basesyntax/service/RegistrationServiceImpl.java

```diff
@@ -2,13 +2,67 @@
 
 import core.basesyntax.dao.StorageDao;
 import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.exception.InvalidDataException;
 import core.basesyntax.model.User;
 
 public class RegistrationServiceImpl implements RegistrationService {
-    private final StorageDao storageDao = new StorageDaoImpl();
+    public static final int ADULT_AGE = 18;
+    public static final int MAX_AGE = 120;
+    public static final int MIN_PASSWORD_LENGTH = 6;
+    public static final int MAX_PASSWORD_AND_LOGIN_LENGTH = 100;
+    public static final int MIN_LOGIN_LENGTH = 2;
+    private final StorageDao storageDao;
+
+    public RegistrationServiceImpl() {
+        storageDao = new StorageDaoImpl();
+    }
 
     @Override
-    public User register(User user) {
-        return null;
+
+    public User register(User user) throws InvalidDataException {
```

Comment:
do we need to indicate that method throws RuntimeException?

Reply:
I think yes, because other dev must know that in this method we can have some exception, and he must do something with it.

Reply:
sorry, we don't need throws RuntimeExeption in method signature.